pom.xml

```diff
@@ -127,6 +127,9 @@
                             <include>**/*Test.java</include>
                             <include>**/*TestSuite.java</include>
                         </includes>
+                        <excludes>
+                            <exclude>org/wildfly/security/ldap/*Test.java</exclude>
```

Comment:
Sorry - one more change.

Now that we are back in a suite lets rename each of the test classes - previously we replaced the suffix 'Test' with 'SuiteChild' that way we do not need a specific exclusion here - also 'SuiteChild' makes it clearer they are part of s suite.

Previously we had some confusion where it looked like tests were not being run when really they were within a suite.